### DISCUSSION TITLE
Does anyone know how to use the key words to link a page?

### DISCUSSION THREAD

**Discussion by ning3739** [quarto-dev/quarto-cli#3546] [Q&A]
I made a home page, and i want to use my keywords to link my other page. 

The [reference](https://quarto.org/docs/websites/#linking) did not explain much about this part.  I still do know what should i do? 

Does anyone have solution? Thanks
![Screenshot 2022-12-02 at 9 44 16 AM](https://user-images.githubusercontent.com/94179273/205156066-e923298b-65b0-4ffb-be26-56750faa466d.png)

**Comment by dragonstyle** [comment]
I'm not sure I understand what you're trying to accomplish - can you say a bit more or provide a sample qmd that might illustrate your goal? You should be able to make a link to another page just using a markdown link to the `qmd` file itself, for example:

```
[Blog](blog.qmd)
```
